plugins/configs/mzML.LCMS_to_MetaboliteAtlas2_MAFileInfo.json

```diff
@@ -0,0 +1,84 @@
+{
```

Comment:
Is it possible to rename this file from
    mzML.LCMS_to_MetaboliteAtlas2_MAFileInfo.json
into
    mzML.LCMS_to_MetaboliteAtlas2.MAFileInfo.json
?
It's not so much required. It's just keeping naming system consistent (take a look at other config file names).